packages/core/admin/admin/src/pages/SettingsPage/pages/ApplicationInfosPage/index.js

```diff
@@ -75,126 +92,128 @@ const ApplicationInfosPage = () => {
+                </Button>
+              ) : null
+            }
+          />
+          <ContentLayout>
+            <Stack spacing={6}>
+              <Box
+                hasRadius
+                background="neutral0"
+                shadow="tableShadow"
+                paddingTop={6}
+                paddingBottom={6}
+                paddingRight={7}
+                paddingLeft={7}
+              >
+                <Stack spacing={5}>
+                  <Typography variant="delta" as="h3">
+                    {formatMessage({
+                      id: 'global.details',
+                      defaultMessage: 'Details',
+                    })}
+                  </Typography>
 
-                <Grid paddingTop={1}>
-                  <GridItem col={6} s={12}>
-                    <Typography variant="sigma" textColor="neutral600">
-                      {formatMessage({
-                        id: 'Settings.application.strapiVersion',
-                        defaultMessage: 'strapi version',
-                      })}
-                    </Typography>
-                    <Typography as="p">v{strapiVersion}</Typography>
-                    <Link
-                      href={
-                        appInfos.communityEdition
-                          ? 'https://discord.strapi.io'
-                          : 'https://support.strapi.io/support/home'
-                      }
-                      isExternal
-                      endIcon={<ExternalLink />}
-                    >
-                      {formatMessage({
-                        id: 'Settings.application.get-help',
-                        defaultMessage: 'Get help',
-                      })}
-                    </Link>
-                  </GridItem>
-                  <GridItem col={6} s={12}>
-                    <Typography variant="sigma" textColor="neutral600">
-                      {formatMessage({
-                        id: 'Settings.application.edition-title',
-                        defaultMessage: 'current plan',
-                      })}
-                    </Typography>
-                    <Typography as="p">
-                      {formatMessage({
-                        id: currentPlan,
-                        defaultMessage: `${
-                          appInfos.communityEdition ? 'Community Edition' : 'Enterprise Edition'
-                        }`,
-                      })}
-                    </Typography>
-                  </GridItem>
-                </Grid>
+                  <Grid paddingTop={1}>
+                    <GridItem col={6} s={12}>
+                      <Typography variant="sigma" textColor="neutral600">
+                        {formatMessage({
+                          id: 'Settings.application.strapiVersion',
+                          defaultMessage: 'strapi version',
+                        })}
+                      </Typography>
+                      <Typography as="p">v{strapiVersion}</Typography>
```

Comment:
I know it is not really related to your PR, but these should be span elements and be wrapped by a single p to make it a bit more accessible. I think ideally this could be a `dl`/ `dt` structure, but that would be too complicated with our DS I think.

Same for the other key/ value pairs.

Would it make sense to change this or a separate PR?

Reply:
interesting, I didn't about about `dl`/`dt`/`dd`- thanks!

I think I would prefer to do in a separate PR, I'll create a jira task!
I'll have a question about the implementation that I'll ask in a comment there :)